unabridged-works-volume-2/original/03-new-old-covenant.adoc

```diff
@@ -1567,11 +1628,10 @@ no days having ever had any real holiness in them one above another;
 but only a figurative, or representative, which the substance, Christ and his Gospel,
 swallows up: for as his day dawns, those things which were the shadows of it fly away.
 
-==== THE SECOND LETTER ANSWERED.
+==== The Second Letter Answered.
 
 [.discourse-part]
-Objection.
-1+++.+++ IT is said to be "`dangerous so to hold forth the spiritual sabbatism,
+_Objection 1+++.+++_ It is said to be "`dangerous so to hold forth the spiritual sabbatism,
```

Comment:
So I took a little liberty here I think....I"m not sure if I was just looking at it wrong at first though. I originally formatted these to be:

[.discourse-part]
Objection.

[.numbered]
1+++.+++ blah blah blah......

but then looking at it in the PDF it looked a little funny so I just made them how they are now. Idk if that's right, but it looked a little better to me in the PDF, although the other way looks okay too.

Reply:
On the same note, just so your mental model is correct, if there is no blank lines between two lines, that means they get joined up with a space in the rendered document.  So, for instance, this asciidoc:

```ASCIIDOC
Objection.
1.
```

Get's rendered as `Objection. 1.`

But, if there is a _blank line_ between them, like this:

```ASCIIDOC
Objection.

1.
```

Then that gets rendered as _two paragraphs_, like this:

```
Objection.

1.
```

So anyway, that's why I said the thing about not introducing a new paragraph by splitting lines apart that didn't _already_ have a blank line between them.